apps/hpc-ftsadmin/src/app/pages/add-edit/flow-edit.tsx

```diff
@@ -0,0 +1,1857 @@
+    fontSize: 11,
+  },
+}));
+
+const initialFormData = {
+  amountUSD: '',
+  amountOriginal: '',
+  exchangeRateUsed: '',
+  keywords: [],
+  flowStatus: '',
+  flowType: { value: 133, displayLabel: 'Standard' },
+  flowDescription: '',
+  firstReported: dayjs().format('DD/MM/YYYY'),
+  decisionDate: null,
+  budgetYear: '',
+  flowDate: null,
+  contributionType: { value: 50, displayLabel: 'Financial' },
+  earmarkingType: '',
+  method: { value: 156, displayLabel: 'Traditional aid' },
+  beneficiaryGroup: '',
+  inactiveReason: '',
+  notes: '',
+  sourceOrganizations: [],
+  sourceLocations: [],
+  sourceUsageYears: [],
+  sourceProjects: [],
+  sourcePlans: [],
+  sourceGoverningEntities: [],
+  sourceGlobalClusters: [],
+  sourceEmergencies: [],
+  destinationOrganizations: [],
+  destinationLocations: [],
+  destinationUsageYears: [],
+  destinationProjects: [],
+  destinationPlans: [],
+  destinationGoverningEntities: [],
+  destinationGlobalClusters: [],
+  destinationEmergencies: [],
+  origCurrency: '',
+  includeChildrenOfParkedFlows: true,
+  reportDetails: [
+    {
+      verified: 'verified',
+      reportSource: 'primary',
+      reporterReferenceCode: '',
+      reportChannel: '',
+      reportedOrganization: { value: '', displayLabel: '' },
+      reportedDate: null,
+      reporterContactInformation: '',
+      sourceSystemRecordId: '',
+      reportFiles: [
+        {
+          title: '',
+          fileName: '',
+          UploadFileUrl: '',
+          size: 0,
+          type: '',
+        },
+      ],
+      reportFileTitle: '',
+      reportUrlTitle: '',
+      reportUrl: '',
+    },
+  ],
+  parentFlow: [],
+  childFlow: [],
+  isParkedParent: false,
+  sources: {},
+};
+
+const initialInputEntries = {
+  amountUSD: null,
+  keywords: null,
+  flowStatus: null,
+  flowType: null,
+  flowDescription: null,
+  contributionType: null,
+  earmarkingType: null,
+  method: null,
+  beneficiaryGroup: null,
+  inactiveReason: null,
+  amountOriginal: null,
+  exchangeRateUsed: null,
+  notes: null,
+  sourceOrganizations: [],
+  sourceLocations: [],
+  sourceUsageYears: [],
+  sourceProjects: [],
+  sourcePlans: [],
+  sourceGoverningEntities: [],
+  sourceGlobalClusters: [],
+  sourceEmergencies: [],
+  destinationOrganizations: [],
+  destinationLocations: [],
+  destinationUsageYears: [],
+  destinationProjects: [],
+  destinationPlans: [],
+  destinationGoverningEntities: [],
+  destinationGlobalClusters: [],
+  destinationEmergencies: [],
+  parentFlow: [],
+  childFlow: [],
+  isParkedParent: false,
+  sources: [],
+};
+
+export default (props: Props) => {
+  const env = getEnv();
+  const { flowId, versionId } = useParams<{
+    flowId: string;
+    versionId: string;
+  }>();
+  const [flowData, setFlowData] = useState<FormValues>(
+    JSON.parse(JSON.stringify(initialFormData))
+  );
+  const [inputEntries, setInputEntries] = useState<InputEntriesType>(
+    JSON.parse(JSON.stringify(initialInputEntries))
+  );
```

Comment:
If you do as the previous comment said, you don't need to explictly define the types here in the `useState()`, Also, remove the JSON functions. When you type things many errors will appear, solve them, they will also help the app to have less bugs.

```suggestion
  const [flowData, setFlowData] = useState(initialFormData);
  const [inputEntries, setInputEntries] = useState(initialInputEntries);
```